Gems/LmbrCentral/Code/include/LmbrCentral/Component/EditorWrappedComponentBase.inl

```diff
@@ -205,6 +207,7 @@ namespace LmbrCentral
         AzToolsFramework::EditorVisibilityNotificationBus::Handler::BusDisconnect();
         AzToolsFramework::Components::EditorComponentBase::Deactivate();
 
+        m_runtimeComponentActive = false;
         m_component.Deactivate();
```

Comment:
nit, I'd reverse these to be symmetric with Activate() (Just in case Deactivate() or Activate() inside the component end up calling something that checks this flag one day)

Reply:
You do it symmetrically in ConfigurationChanged(), so that's an extra reason to do that here too.